kafka-daq-report/src/main.rs

```diff
@@ -103,20 +126,29 @@ enum Event<I> {
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    let args = Cli::parse();
+    tracing_subscriber::fmt::init();
```

Comment:
Have you tested what happens in "DAQ trace" mode? The TUI will likely not function correctly with this line here.

Reply:
Ah I see, this does cause problems when running with `RUST_LOG=trace`.